packages/addons/service/mariadb/package.mk

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mariadb"
+PKG_VERSION="10.3.11"
+PKG_REV="100"
+PKG_SHA256="211655b794c9d5397ba3be6c90737eac02e882f296268299239db47ba328f1b2"
+PKG_LICENSE="GPL2"
+PKG_SITE="https://mariadb.org/"
+PKG_URL="https://downloads.mariadb.org/interstitial/${PKG_NAME}-${PKG_VERSION}/source/${PKG_NAME}-${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_HOST="toolchain ncurses:host"
+PKG_DEPENDS_TARGET="toolchain ncurses libaio openssl zlib bzip2 lzo binutils systemd libxml2 mariadb:host"
```

Comment:
```suggestion
PKG_DEPENDS_TARGET="toolchain binutils bzip2 libaio libxml2 lzo mariadb:host ncurses openssl systemd zlib"
```
just ocd

Reply:
mariadb:host should be last so first host part is build and then target. It looks nicer when checking build log.

Reply:
makes sense, I changed it